renku/core/models/jsonld.py

```diff
@@ -553,6 +553,8 @@ def from_jsonld(
     @classmethod
     def from_yaml(cls, path, client=None, commit=None):
         """Return an instance from a YAML file."""
+        assert False, 'this should not be called.'
```

Comment:
isn't it still needed for the classes that have not been migrated to calamus?

Reply:
No. Dataset and Project are the only classes that use this method for serialisation.